src/modules/authentication/ClaimsChallenge.ts

```diff
@@ -0,0 +1,60 @@
+import { authenticationWrapper } from './index';
+import { configuration } from './msal-app';
+import IClaimsChallenge from './interfaces/IClaimsChallenge';
+import { IQuery } from '../../types/query-runner';
+export class ClaimsChallenge implements IClaimsChallenge {
+  private static instance: ClaimsChallenge;
+
+  public static getInstance(){
+    if (!ClaimsChallenge.instance) {
+      ClaimsChallenge.instance = new ClaimsChallenge();
+    }
+    return ClaimsChallenge.instance;
+  }
+  public handleClaimsChallenge(responseHeaders: Headers, sampleQuery: IQuery){
+    const account = authenticationWrapper.getAccount();
```

Comment:
Currently the `ClaimsChallenge` depends on `AuthenticationWrapper` which also depends on `ClaimsChallenge`. This looks like cyclic dependency. 
This is the only place where the authenticationWrapper is needed, so it might just make life easier to pass in the account as a parameter or for the claimsChallenge to derive the account presence state